pyproject.toml

```diff
@@ -49,15 +49,16 @@ artifacts = [
   "*.pyccel",
 ]
 include = [
-  "include",
+  "pyccel/extensions/STC/include",
   "*.py",
   "*.pyh",
   "*.pyccel",
   "pyccel/parser/grammar/*.tx",
   "pyccel/stdlib/*/*.h",
   "pyccel/stdlib/*/*.c",
   "pyccel/stdlib/*/*.f90",
-  "pyccel/extensions/STC/include/*/*.c"
+  "pyccel/extensions/STC/include/*/*.c",
+  "pyccel/extensions/gFTL/include/v2"
```

Comment:
Can we reorder this to group the extensions together please

Reply:
Done : 2cb3c6e